packages/plugin-core/src/test/suite-integ/GoToSibling.test.ts

```diff
@@ -182,6 +185,27 @@ suite("GoToSibling", () => {
       });
     });
 
+    describeSingleWS("no active dendron note", {}, () => {
+      test("Warning message should appear", async () => {
+        await VSCodeUtils.closeAllEditors();
+        // Create a file that is not a Dendron note and show it on editor
+        const workspaceRootPath =
```

Comment:
would not use `workspaceFolders` since this will write it to the code workspace. instead, get `wsRoot` from `ExtensionProvider.getEngine().wsRoot`